theano/gof/op.py

```diff
@@ -904,6 +897,9 @@ def make_thunk(self, node, storage_map, compute_map, no_recycling):
         no_recycling
             List of variables for which it is forbidden to reuse memory
             allocated by a previous call.
+        impl
+            Currently, None, 'c' or 'py'. If 'c' or 'py' we will only try
+            that version of the code.
```

Comment:
Not sure if you care, but note that changing the interface like this breaks existing code such as the cuda-convnet wrappers (https://github.com/lisa-lab/pylearn2/blob/5513089/pylearn2/sandbox/cuda_convnet/filter_acts.py#L90). Just noticed that by chance. (In linker/vm.py you call make_node with impl=impl, and this breaks if make_node does not expect an impl argument at all.)